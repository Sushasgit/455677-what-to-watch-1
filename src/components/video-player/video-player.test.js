import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

it(`VideoPlayer run correctly`, () => {
  const tree = renderer
    .create(<VideoPlayer
      src={``}
      isPlaying={true}
      posterUrl={``}
      muted={true}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
