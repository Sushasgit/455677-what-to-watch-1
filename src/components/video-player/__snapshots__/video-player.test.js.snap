// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer run correctly 1`] = `
<video
  className="small-movie-card__video"
  muted={true}
  poster=""
/>
`;
